### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed Changes**:
- remove duplicate stories
- remove utter_tryout block and replace with transition to step 2
- remove feedback form on ultimate fallback policy
- show community events which happen on the same day
- fix feedback stories (utter_thumpsup)

### CODE REVIEW

**Comment by wochinge** [comment]
this does not include the nicely formatted entities thing. I'll do that separately

File: data/core/closetheloop.md [hunk 1]
```diff
@@ -463,23 +432,6 @@
...
-    - utter_first_bot_with_rasa
-* deny
-    - action_set_onboarding
-    - slot{"onboarding":false}
-    - utter_ask_which_product
-* how_to_get_started{"product":"core"}
-    - slot{"product":"core"}
-    - utter_core_tutorial
-    - utter_anything_else
-* how_to_get_started{"product":"nlu"}
```
Comment (akelad) [inline data/core/closetheloop.md]: just wondering why we're removing this one?

File: data/core/closetheloop.md [hunk 2]
```diff
@@ -725,38 +677,6 @@
...
-## Story from conversation with 7830abb04e1c49809d89b0d420443928 on November 19th 2018
-* greet
-    - action_greet_user
-* ask_whoisit
-    - action_chitchat
-* out_of_scope
-    - utter_out_of_scope
-* ask_whatspossible
-    - action_chitchat
-* how_to_get_started{"product":"nlu"}
```
Comment (akelad) [inline data/core/closetheloop.md]: also these two?

File: data/core/closetheloop.md [hunk 3]
```diff
@@ -788,52 +708,7 @@
 * affirm
     - utter_explain_nlu
     - utter_explain_core
-    - utter_tryout
```
Comment (akelad) [inline data/core/closetheloop.md]: shouldn't this be `utter_direct_step2`?

File: data/core/closetheloop.md [hunk 4]
```diff
@@ -788,52 +708,7 @@
...
-
-## Story from conversation with 67a8696eb5894b25a800b6cbd7a695bb on December 15th 2018
-* get_started_step1
-    - action_greet_user
-    - slot{"shown_privacy":true}
-* ask_howdoing
-    - action_chitchat
-* ask_whoisit
-    - action_chitchat
-* enter_data
```
Comment (akelad) [inline data/core/closetheloop.md]: why are we removing this?

File: data/core/closetheloop.md [hunk 5]
```diff
@@ -788,52 +708,7 @@
...
-    - utter_built_bot_before
-* deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
-* affirm
-    - utter_explain_nlu
-    - utter_explain_core
-    - utter_tryout
-* how_to_get_started{"product":"nlu"}
```
Comment (akelad) [inline data/core/closetheloop.md]: same question

File: data/core/closetheloop.md [hunk 6]
```diff
@@ -862,7 +737,6 @@
     - utter_not_sure
     - utter_possibilities
 * ask_faq_languages
-    - action_store_problem_description
     - slot{"problem_description":"what languages do you support?"}
```
Comment (akelad) [inline data/core/closetheloop.md]: slot events also need to be removed

File: data/core/closetheloop.md [hunk 7]
```diff
@@ -881,7 +755,6 @@
...
```
Comment (akelad) [inline data/core/closetheloop.md]: same here

File: data/core/faqs.md [hunk 8]
```diff
@@ -807,39 +774,10 @@
...
-    - utter_explain_nlucore
-* affirm OR how_to_get_started{"product":"stack"}
-    - utter_explain_nlu
-    - utter_explain_core
-    - utter_tryout
+    - utter_direct_to_step2
 * ask_faq_platform OR ask_faq_languages OR ask_faq_tutorialcore OR ask_faq_tutorialnlu OR ask_faq_opensource OR ask_faq_voice OR ask_faq_slots OR ask_faq_channels OR ask_faq_differencecorenlu OR ask_faq_python_version OR ask_faq_community_size OR ask_faq_what_is_forum
     - action_faqs
-    - utter_tryout
-* deny
```
Comment (akelad) [inline data/core/faqs.md]: i see we've gotten rid of the `deny` everywhere -- what if a user does say "no"?

File: data/core/get_started.md [hunk 9]
```diff
@@ -709,24 +534,3 @@
...
-* affirm
-    - utter_ask_migration
-* deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
-* affirm OR how_to_get_started{"product":"stack"}
-    - utter_explain_nlu
-    - utter_explain_core
-    - utter_tryout
```
Comment (akelad) [inline data/core/get_started.md]: and this one?

File: data/core/stories.md [hunk 10]
```diff
@@ -452,86 +452,6 @@
     - utter_great
     - utter_anything_else
 
-## just sales
```
Comment (akelad) [inline data/core/stories.md]: why?

File: data/core/stories.md [hunk 11]
```diff
@@ -452,86 +452,6 @@
...
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_store_sales_info
-    - slot{"data_stored": true}
-    - utter_confirm_salesrequest
-    - utter_ask_feedback
-* feedback{"feedback_value": "positive"}
-    - slot{"feedback_value": "positive"}
-    - utter_great
-    - utter_anything_else
```
Comment (akelad) [inline data/core/stories.md]: also why

File: data/core/stories.md [hunk 12]
```diff
@@ -627,58 +547,6 @@
...
-
```
Comment (akelad) [inline data/core/stories.md]: why?

File: domain.yml [hunk 13]
```diff
@@ -511,6 +509,11 @@ templates:
       buttons:
         - title: "Continue"
           payload: "/get_started_step3"
+  utter_direct_to_step2:
+    - text: "If you have no questions, you can click the button below to go to the next step once you're ready. Or when you're finished asking questions, you can say 'take me to the next step' at any point"
```
Comment (akelad) [inline domain.yml]: let's leave out the "if you have no questions" and "take me to the next step thing" -- i think it only really makes sense to have that in step2

File: data/core/closetheloop.md (hunk 2 above)
Comment (wochinge) [reply data/core/closetheloop.md]: second is a duplicate +f ## Story from conversation with 7830abb04e1c49809d89b0d420443928 on November 19th 2018 (still there)

**Comment by akelad** [comment]
@wochinge i also setup testing of the model on travis now -- seems like there's some problems with the stories still, can you take a look?

File: data/core/stories.md (hunk 12 above)
Comment (wochinge) [reply data/core/stories.md]: duplicate of ## newsletter then sales

File: data/core/closetheloop.md (hunk 2 above)
Comment (wochinge) [reply data/core/closetheloop.md]: first is a duplicate of ## Story from conversation with dominik on November 19th 2018

File: data/core/faqs.md (hunk 8 above)
Comment (akelad) [reply data/core/faqs.md]: well we don't need to, i just thought it was weird we were getting rid of these haha

Comment (wochinge) [reply data/core/faqs.md]: So should I re-add it or not? :-D

File: data/core/closetheloop.md (hunk 3 above)
Comment (wochinge) [reply data/core/closetheloop.md]: it is, bit weirdly shown by the diff tool

File: data/core/stories.md (hunk 10 above)
Comment (wochinge) [reply data/core/stories.md]: duplicates

**Comment by wochinge** [comment]
@akelad fixed the stories, now it seems to be a new error which seems not to be related to the stories

File: data/core/faqs.md (hunk 8 above)
Comment (akelad) [reply data/core/faqs.md]: let's leave it for now

**Comment by akelad** [comment]
@wochinge feel free to merge this, the build is failing because it can't plot the confmat, and i'm not 100% sure how to fix this on travis

**Pull request merged**